Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt

```diff
@@ -65,49 +66,34 @@ class ScheduleAdapterItem internal constructor(val itemData: Schedule.ScheduleRo
                                 holder: ScheduleAdapterItem.ViewHolder,
                                 position: Int,
                                 payloads: List<*>) {
+        holder.sessionLayout.visibility = View.VISIBLE
+        holder.avatar.visibility = View.VISIBLE
+        holder.bigTitle.visibility = View.GONE
+
+        holder.title.text = itemData.talkTitle
+        holder.time.text = String.format("%s - %s", itemData.startTime, itemData.endTime)
+        holder.speaker.text = itemData.speakerNames?.joinToString(separator = ", ")
+        holder.room.text = itemData.room
+
+        holder.speakerCount.visibility = if (itemData.speakerCount > 1) View.VISIBLE else View.GONE
+        holder.speakerCount.text = String.format("+%d", itemData.speakerCount - 1)
+
+        val context = holder.title.context
+
+        Glide.with(context)
+                .load(itemData.photoUrlMap?.get(itemData.primarySpeakerName))
+                .transform(CircleTransform(context))
+                .crossFade()
+                .into(holder.avatar)
+
+        val userAgendaRepo = UserAgendaRepo.getInstance(holder.bookmarkIndicator.context)
+        holder.bookmarkIndicator.visibility = if (userAgendaRepo.isSessionBookmarked(itemData.id))
+            View.VISIBLE
+        else
+            View.INVISIBLE
+
+        addBackgroundRipple(holder)
 
-        if (!itemData.hasSpeaker()) {
```

Comment:
This was for the "Lunch" and "Registration" Sessions FYI... those are still missing from the slidesup data..

Reply:
I cn put this back in. Didnt know